mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.js

```diff
@@ -625,6 +657,14 @@ export class MetricsPlotPanel extends React.Component {
               runDisplayNames={runDisplayNames}
               metricKeys={selectedMetricKeys}
             />
+            <Button className='csv-button-metrics' onClick={this.onDownloadCsv}>
```

Comment:
<img width="2981" alt="Screen Shot 2022-06-21 at 14 49 52" src="https://user-images.githubusercontent.com/17039389/174726973-5ee9bec0-7660-4d56-b0df-8137ea2d1ee1.png">

Reply:
@harupy can you try pulling and visualizing it again? - I changed it in one of the last commits (in mine it is on top of the metrics table and on the left)

<img width="1369" alt="Screenshot 2022-06-22 at 18 38 07" src="https://user-images.githubusercontent.com/97888102/175090386-de16c468-1131-4537-b5ee-67118c5c0ece.png">
:

Reply:
does this look better @BenWilson2 @harupy ?

<img width="1278" alt="Screenshot 2022-06-22 at 18 50 41" src="https://user-images.githubusercontent.com/97888102/175093796-81195e8a-a549-4daa-8544-0613782b081a.png">

Reply:
awesome thanks!

Reply:
Maybe it's just me, but it looks like the button downloads the metric summary table as CSV, but it actually doesn't.